src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -205,14 +205,19 @@ void FrontEnd::translate_graph(const ov::frontend::InputModel::Ptr& model,
 
         if (port_type == "none") {
             for (const auto& node_output : ng_op_map[operation_name]) {
-                results.push_back(std::make_shared<ov::opset8::Result>(node_output));
+                auto result_node = std::make_shared<ov::opset8::Result>(node_output);
+                // for alignment with legacy TensorFlow frontend, we must use OpName:0 for Result node
+                set_node_name(operation_name + ":" + std::to_string(port_index), result_node);
```

Comment:
`set_node_name` function also does add ":port" internally for each output tensor, is it expected? While Results are special nodes maybe it do not use `set_node_name` for Results, but manually set friendly name and tensor names if needed?

Reply:
`set_node_name` function sets a name for node + set names for output tensors of this node. Since `Result` node does not have outputs, it sets only node name. It should be fine for it.
Generally, now it does not affect API 2.0 inference because it does not rely on `Result` node name at all. I just set more meaningful name for it